packages/salesforcedx-vscode-replay-debugger/src/states/frameEntryState.ts

```diff
@@ -19,11 +20,25 @@ export class FrameEntryState extends FrameState implements DebugLogState {
 
   public handle(logContext: LogContext): boolean {
     const sourceUri = logContext.getUriFromSignature(this._signature);
+    const frame = new ApexDebugStackFrameInfo(
```

Comment:
Assigned it a unique id and pushed that into the frame itself. This id is used to retrieve the stack frame info object where it can access the variables.